charts/camunda-platform-8.8/values-digest.yaml

```diff
@@ -0,0 +1,35 @@
+console:
```

Comment:
We don't need to override everything, just the component that requires the digest.

Reply:
I removed console and web-modeler. The rest are needed from my testing.
I would prefer to keep web-modeler and console for the sake of completeness.